modules/cli/pages/mctimings.adoc

```diff
@@ -93,501 +73,331 @@ If using a secure connection (the `-s` flag) then this should be 11211.
-[ 800 -  809]us ( 98.78%)     5 |
-[ 810 -  819]us ( 98.80%)     7 |
-[ 820 -  829]us ( 98.81%)     1 |
-[ 830 -  839]us ( 98.83%)     5 |
-[ 840 -  849]us ( 98.84%)     3 |
-[ 850 -  859]us ( 98.84%)     1 |
-[ 860 -  869]us ( 98.85%)     2 |
-[ 880 -  889]us ( 98.87%)     5 |
-[ 890 -  899]us ( 98.88%)     2 |
-[ 900 -  909]us ( 98.90%)     5 |
-[ 910 -  919]us ( 98.91%)     3 |
-[ 920 -  929]us ( 98.93%)     3 |
-[ 940 -  949]us ( 98.94%)     3 |
-[ 950 -  959]us ( 98.95%)     2 |
-[ 960 -  969]us ( 98.96%)     3 |
-[ 970 -  979]us ( 98.97%)     4 |
-[ 980 -  989]us ( 98.99%)     3 |
-[ 990 -  999]us ( 99.01%)     6 |
-[   1 -    1]ms ( 99.57%)   138 | #
-[   2 -    2]ms ( 99.74%)    42 |
-[   3 -    3]ms ( 99.82%)    18 |
-[   4 -    4]ms ( 99.89%)    17 |
-[   5 -    5]ms ( 99.92%)     8 |
-[   6 -    6]ms ( 99.94%)     6 |
-[   7 -    7]ms ( 99.96%)     3 |
-[   8 -    8]ms ( 99.96%)     1 |
-[   9 -    9]ms ( 99.96%)     1 |
-[  11 -   11]ms ( 99.97%)     1 |
-[  12 -   12]ms ( 99.98%)     2 |
-[  13 -   13]ms ( 99.98%)     1 |
-[  18 -   18]ms ( 99.98%)     1 |
-[  27 -   27]ms ( 99.99%)     1 |
-[  28 -   28]ms ( 99.99%)     1 |
-[  41 -   41]ms (100.00%)     1 |
-[  50 -  499]ms (100.00%)     1 |
-Total: 24576 operations
-The following data is collected for "GET_CMD_TIMER"
-[  50 -   59]us ( 53.16%)  1491 | ############################################
-[  60 -   69]us ( 81.85%)   805 | #######################
-[  70 -   79]us ( 88.06%)   174 | #####
-[  80 -   89]us ( 90.66%)    73 | ##
-[  90 -   99]us ( 92.69%)    57 | #
-[ 100 -  109]us ( 94.55%)    52 | #
-[ 110 -  119]us ( 95.44%)    25 |
-[ 120 -  129]us ( 96.08%)    18 |
-[ 130 -  139]us ( 96.47%)    11 |
-[ 140 -  149]us ( 97.04%)    16 |
-[ 150 -  159]us ( 97.22%)     5 |
-[ 160 -  169]us ( 97.43%)     6 |
-[ 170 -  179]us ( 97.83%)    11 |
-[ 180 -  189]us ( 98.07%)     7 |
-[ 190 -  199]us ( 98.22%)     4 |
-[ 200 -  209]us ( 98.40%)     5 |
-[ 210 -  219]us ( 98.43%)     1 |
-[ 220 -  229]us ( 98.57%)     4 |
-[ 230 -  239]us ( 98.65%)     2 |
-[ 240 -  249]us ( 98.68%)     1 |
-[ 250 -  259]us ( 98.72%)     1 |
-[ 260 -  269]us ( 98.82%)     3 |
-[ 270 -  279]us ( 98.89%)     2 |
-[ 280 -  289]us ( 98.93%)     1 |
-[ 290 -  299]us ( 98.97%)     1 |
-[ 300 -  309]us ( 99.04%)     2 |
-[ 320 -  329]us ( 99.07%)     1 |
-[ 350 -  359]us ( 99.14%)     2 |
-[ 360 -  369]us ( 99.22%)     2 |
-[ 390 -  399]us ( 99.25%)     1 |
-[ 400 -  409]us ( 99.32%)     2 |
-[ 430 -  439]us ( 99.36%)     1 |
-[ 450 -  459]us ( 99.39%)     1 |
-[ 500 -  509]us ( 99.43%)     1 |
-[ 510 -  519]us ( 99.50%)     2 |
-[ 630 -  639]us ( 99.54%)     1 |
-[ 640 -  649]us ( 99.57%)     1 |
-[ 750 -  759]us ( 99.61%)     1 |
-[ 800 -  809]us ( 99.68%)     2 |
-[ 860 -  869]us ( 99.71%)     1 |
-[ 870 -  879]us ( 99.75%)     1 |
-[   1 -    1]ms ( 99.93%)     5 |
-[  32 -   32]ms ( 99.96%)     1 |
-[  39 -   39]ms (100.00%)     1 |
-Total: 2805 operations
+[  0.00 -   6.00]us (0.0000%)	   2|
```

Comment:
We would now print the legend too before the first histogram is rendered. As the -v flag is set.